src/utils/index.js

```diff
@@ -3,7 +3,7 @@ import parse from "rehype-parse";
 import format from "rehype-format";
 import stringify from "rehype-stringify";
 
-const KS_RE = /{{([^}]*)}}/g;
+const KS_RE = /{{2}([^}].+?)}{2}/g;
```

Comment:
This is to address https://github.com/mdn/markdown/runs/10601035645

Where `{{{{{{{{{{{{{{{{{{` would be an expensive input string for the regex to match.

Only matching two `{` and `}` characters (`{{2}` and `}{2}`) seems to be simplest fix I can think of.

Reply:
CCing @queengooborg -> you did some work on the tests here so you may have better context